server/odc-server/src/main/resources/data.sql

```diff
@@ -758,4 +758,10 @@ INSERT INTO config_system_configuration(`key`, `value`, `description`) VALUES('o
 INSERT INTO config_system_configuration ( `key`, `value`, `description` )
 VALUES
   ( 'odc.task.async.index-change-max-timeout-millis', '432000000', 'If the change content of the database change task involves time-consuming index change operations, the timeout period for the automatically modified database change task, unit: milliseconds. Default value is 5 days.' )
-  ON DUPLICATE KEY UPDATE `id` = `id`;
\ No newline at end of file
+  ON DUPLICATE KEY UPDATE `id` = `id`;
+
```

Comment:
add a configuration like `odc.iam.auth.method` to control which method should we use~
candidates:
1. jsessionid
2. jwt